src/QSwagGenerator/Generators/SchemaGenerator.cs

```diff
@@ -251,7 +252,7 @@ internal static bool IsParameterRequired(ParameterInfo parameter)
             if (isNullable) return false;
             return true;
 
-            return parameter.ParameterType.GetTypeInfo().IsValueType;
+            //return parameter.ParameterType.GetTypeInfo().IsValueType;
```

Comment:
commented this out since it was generating a compiler warning, it is not reachable due to the return above.